xlators/cluster/afr/src/afr-self-heald.c

```diff
@@ -1658,7 +1576,7 @@ afr_xl_op(xlator_t *this, dict_t *input, dict_t *output)
             /* this case has been handled in glfs-heal.c */
             break;
         case GF_SHD_OP_SPLIT_BRAIN_FILES:
```

Comment:
nit: move this below line 1575?

Reply:
Yes, we can.
Will wait for other reviews and the regression to complete before resubmitting.

Reply:
Done.